admin/services/data-access/redis-cache.service.js

```diff
@@ -31,6 +31,7 @@ const redisCacheService = {}
 redisCacheService.get = async key => {
   redisConnect()
   try {
+    logger.info(`REDIS (get): retrieving ${key}`)
```

Comment:
I am probably reading this out of context, but I think that the Redis logger calls should exist at the same level as the SQL ones, `debug`?

Reply:
all changed